foreman_openscap.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |s|
   s.test_files = Dir["test/**/*"]
 
   s.add_dependency 'deface', '~> 0'
-  s.add_dependency 'scaptimony', '~> 0'
+  s.add_dependency 'scaptimony', ['>= 0.3.0', '< 0.4.0']
```

Comment:
definitely for non-rubyist, IMHO rubyists will find `['>= 0.3.0', '< 0.4.0']` weird and will start to think why you used this... anyway it's a small detail